src/bin/vip-import-sql.js

```diff
@@ -180,13 +212,187 @@ const examples = [
 	},
 ];
 
+const promptToContinue = async ( {
+	launched,
+	formattedEnvironment,
+	track,
+	domain,
+} ): Promise<void> => {
+	console.log();
+	const promptToMatch = domain.toUpperCase();
+	const promptResponse = await prompt( {
+		type: 'input',
+		name: 'confirmedDomain',
+		message: `You are about to import the above tables into a ${
+			launched ? 'launched' : 'un-launched'
+		} ${ formattedEnvironment } site ${ chalk.yellow( domain ) }.\nType '${ chalk.yellow(
+			promptToMatch
+		) }' (without the quotes) to continue:\n`,
+	} );
+
+	if ( promptResponse.confirmedDomain !== promptToMatch ) {
+		await track( 'import_sql_unexpected_tables' );
+		exit.withError( 'The input did not match the expected environment label. Import aborted.' );
+	}
+};
+
+export type validationsAndGetTableNamesInputType = {
+	skipValidate: boolean,
+	appId: number,
+	envId: number,
+	fileNameToUpload: string,
+};
+
+export async function validationsAndGetTableNames( {
```

Comment:
Super minor, but this reads odd to me.  I'd suggest either:

```suggestion
export async function validateAndGetTableNames( {
```

or something like `applyValidationsAndGetTableNames`